Other/ExtractingPartitions.md

```diff
@@ -18,44 +18,31 @@
 
 ## Getting your current boot slot (A/B)
 
-Boot to Android™, on your PC open a command prompt window and type:
+Assuming your Surface Duo is booted to Android™, plugged into to your PC:
```

Comment:
nit: plug it into your PC

Reply:
also mention to start the command into a command prompt, users may not already be in one

Reply:
also the sentence sounds a bit wrong to me.

Maybe more "Plug your Surface Duo into your PC. Assuming your device is currently booted into Android™️, type the following command into a command prompt:"